### DISCUSSION TITLE
Problems converting book using Kindle 1.17, Calibre 5.8.1 and DeDrm 7.0.3

### DISCUSSION THREAD

**Discussion by Saccadio** [apprenticeharper/DeDRM_tools#1464] [Q&A]
I just tried using Calibre 5.8.1 to import a book from my Windows 10 downloaded using Kindle  1.17. The book can be read fine on my Windows Kindle. But it it is scrambled when trying to open it in Calibre after importing. There are no error messages or anything. The book is https://www.amazon.com/Fascia-Function-Medical-Applications-Lesondak-ebook-dp-B08F2V8YSN/dp/B08F2V8YSN/ref=mt_other?_encoding=UTF8&me=&qid=

I bought two other books last days of December and converted them using the same setup. So, what can I do to make it work with this new book?

**Comment by ElleKayEm** [comment]
That's a print replica book, a PDF in an Amazon wrapper.  I believe you'll need to go back to calibre 4.23 and DeDRM 6.8.1 to decrypt that.  Then I suggest using the Kindle Unpack plugin to extract the pdf.

**Reply by Saccadio** [reply]
Thanks very much. It worked very well.

Others have argued why we need Calibre to support to versions installed on the same machine. This has always been countered, but the last month's problems with version 4.23 vs 5.xx shows that this really would spare us from installing Calibre on two different machines as it is necessary now.

**Reply by ElleKayEm** [reply]
If you're downloading a book with the Kindle app, it needs to be on the same machine and user as calibre with DeDRM in order for DeDRM to retrieve the key needed to unlock that book.  This has always been the case.

People have suggested running a portable version of calibre 4.23 and a full install calibre 5.x (or vice versa) on the same Windows machine.  You could also have a setup in another user on the same machine.  Personally, I'm just keeping Python 2 calibre 4.23/DeDRM 6.8 until there's a full release of DeDRM for Python 3 calibre.  There's no new feature in later versions of calibre that beats being able to reliably remove DRM.
